### PR TITLE
feat: add assets tab

### PR DESCRIPTION
**Pull request by meelrossi**
Add new assets tab and FF

### CODE REVIEW

File: src/modules/features/selectors.ts [hunk 1]
```diff
@@ -4,8 +4,16 @@ import { RootState } from '../reducer'
 import { FeatureName } from './types'
 
 export const getIsCreationsTabEnabled = (state: RootState) => {
+  return false
```
Comment (LautaroPetaccio) [inline src/modules/features/selectors.ts:7]: A wild return has appeared!
```suggestion
  return false
```

File: src/components/Pages/MainPage/MainPage.tsx [hunk 2]
```diff
@@ -21,39 +22,42 @@ import { Props } from './MainPage.types'
...
 
   const selectedTab = useMemo(() => {
     const tab = params.tab
-    switch (tab) {
-      case AccountTabs.CREATIONS:
-        return AccountTabs.CREATIONS
-      default:
-        return AccountTabs.OVERVIEW
-    }
+    return tab || AccountTabs.OVERVIEW
```
Comment (LautaroPetaccio) [inline src/components/Pages/MainPage/MainPage.tsx:41]: As it doesn't have a select anymore, what about changing the way we write the selected tab as:
```
const selectedTab = params.tab ?? AccountsTabs.OVERVIEW
```

File: src/components/Pages/MainPage/MainPage.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      isAssetsTabEnabled={false}
+      isBlocked={false}
+      {...props}
+    />
+  )
+}
+
+let screen: RenderResult
+
+describe('when creations tab FF is enabled', () => {
```
Comment (LautaroPetaccio) [inline src/components/Pages/MainPage/MainPage.spec.tsx:42]: If I'm not wrong, the article might be missing:
```suggestion
describe('when the creations tab FF is enabled', () => {
```

**Review by LautaroPetaccio — COMMENTED** [review]
The PR looks great! I've left some minor comments to be solved.

File: src/modules/features/selectors.ts (hunk 1 above)
Comment (meelrossi) [reply src/modules/features/selectors.ts:7]: my baddd i was testing combinations and missed it 😬

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**